MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include requirements.txt
 include scripts/*
 include protobuf/*
+include-recursive mediachain/translation *.json
```

Comment:
the docs suggest that this is `recursive-include`? https://docs.python.org/2/distutils/sourcedist.html